schemas/common.json

```diff
@@ -718,6 +718,85 @@
             "items": {
                 "type": "string"
             }
+        },
+        "monit_mail_server": {
+            "type": "string",
+            "title": "Monit Mail Server",
+            "description": "Mail server to be used to receive monit alerts via email",
```

Comment:
Use something similar to the health report wording. We don't want to give them the impression that they need to specify the name of the server where their email client is running. Something like, "Address of the SMTP server to be used when mailing VSD monit alerts"